scripts/resetdb.sh

```diff
@@ -57,11 +62,16 @@ main() {
   if [ -z ${REPLY+x} ] || [[ $REPLY =~ ^[Yy]$ ]]
   then
       echo "Resetting DB..."
-      mysql "${FLAGS[@]}" -e "DROP DATABASE IF EXISTS ${DB_NAME};"
-      mysql "${FLAGS[@]}" -e "CREATE DATABASE ${DB_NAME};"
-      mysql "${FLAGS[@]}" -e "CREATE USER IF NOT EXISTS ${DB_NAME}@'localhost' IDENTIFIED BY 'zaphod';"
-      mysql "${FLAGS[@]}" -e "GRANT ALL ON ${DB_NAME}.* TO ${DB_NAME}@'localhost'"
-      mysql "${FLAGS[@]}" -D ${DB_NAME} < ${TRILLIAN_PATH}/storage/mysql/storage.sql
+      mysql "${FLAGS[@]}" -e "DROP DATABASE IF EXISTS ${DB_NAME};" || \
+        die "Error: Failed to drop database."
```

Comment:
Maybe echo the various parameters (e.g. "Failed to drop database ${DB_NAME}") throughout?

Reply:
Done.